01_prepare_host.sh

```diff
@@ -37,10 +37,20 @@ if [ "$KUBECTL_LOCAL" != "$KUBECTL_LATEST" ]; then
     sudo mv kubectl "${KUBECTL_PATH}"
 fi
 
-if ! command -v kustomize 2>/dev/null ; then
-    curl -Lo kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F"${KUSTOMIZE_VERSION}"/kustomize_kustomize."${KUSTOMIZE_VERSION}"_linux_amd64
-    chmod +x kustomize
-    sudo mv kustomize /usr/local/bin/.
+if [ "${EPHEMERAL_CLUSTER}" == "minikube" ]; then
+  if ! command -v kustomize 2>/dev/null ; then
+      curl -Lo kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F"${KUSTOMIZE_VERSION}"/kustomize_kustomize."${KUSTOMIZE_VERSION}"_linux_amd64
+      chmod +x kustomize
+      sudo mv kustomize /usr/local/bin/.
+  fi
+fi
```

Comment:
why are you installing kustomize only if it is a Minikube cluster ? You should install it always. However you should install minikube and initialize minikube only if you are using Minikube

Reply:
Good finding, copy paste bug. Fixed

Reply:
Fixed